app/src/main/java/com/example/wnews/views/auth/AuthPresenter.kt

```diff
@@ -14,6 +15,29 @@ import retrofit2.Response
 
 class AuthPresenter(var sharedPref: SharedPreferences, val view: AuthView?) {
 
+    fun validateUserMail(userMail: String): Boolean {
+
+        return FormatUtils().EMAIL_ADDRESS_PATTERN.matcher(userMail).matches()
+    }
+
+    fun isUserMailEmpty(userMail: String): Boolean {
+
+        if (userMail.isEmpty()) {
```

Comment:
This can be simplified like this:

```kotlin
fun isUserMailEmpty(userMail: String): Boolean {
    return userMail.isEmpty()
}
```
And if I recall correctly, in Kotlin this would be possible too:

```kotlin
fun isUserMailEmpty(userMail: String) = userMail.isEmpty()
```